src/Model/Fulfilment/OrderLine.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Model\Fulfilment;
+
+use MyParcelNL\Sdk\src\Model\BaseModel;
+
+class OrderLine extends BaseModel
+{
+    /**
+     * @var array
+     */
+    private $instructions;
+
+    /**
+     * @var int|null
+     */
+    private $price;
+
+    /**
+     * @var int|null
+     */
+    private $price_after_vat;
+
+    /**
+     * @var \MyParcelNL\Sdk\src\Model\Fulfilment\Product
+     */
+    private $product;
+
+    /**
+     * @var int|null
+     */
+    private $quantity;
+
+    /**
+     * @var string|null
+     */
+    private $uuid;
+
+    /**
+     * @var int|null
+     */
+    private $vat;
+
+    /**
+     * @param  array $data
+     */
+    public function __construct(array $data = [])
+    {
+        $this->uuid            = $data['uuid'] ?? null;
+        $this->price           = $this->intOrNull($data['price'] ?? null);
+        $this->vat             = $this->intOrNull($data['vat'] ?? null);
```

Comment:
vat_amount (to distinguish clearly from vat_percentage or something)?

Reply:
Ik ga geen namen wijzigen, dit is allemaal 1 op 1 hetzelfde als in de api